app/routes/_libraries.virtual.$version.index.tsx

```diff
@@ -396,6 +396,7 @@ export default function RouteComp() {
           <div className="flex flex-wrap gap-2 justify-center">
             {(
               [
+                { label: 'Lit', value: 'lit' },
```

Comment:
We should probably follow the same ordering as Table and place the label in the right position in the dropdown.

https://tanstack.com/table/latest/docs/introduction

Reply:
No problem, fixed it, first attempt was simply alphabetically but that makes sense